### PR TITLE
Addresses #54 Add MAS tag to 'blood'

### PR DESCRIPTION
**Pull request by ghost**
Addresses #54 Add Multicellular_anatomical_structure tag to 'blood'.

### CODE REVIEW

File: config/prod/neo4j2owl-config.yaml [hunk 1]
```diff
@@ -211,6 +211,7 @@ neo_node_labelling:
     label: diaphragm
   - classes:
       - UBERON:0010000
+      - UBERON:0000178
```
Comment (ubyndr) [inline config/prod/neo4j2owl-config.yaml:214]: Is this enough to add the tag to subClasses as well or do we need some DL query in here?

Comment (ghost) [reply config/prod/neo4j2owl-config.yaml:214]: We have other examples, that I think are also meant to pull in subclasses. E.g.,

```
  - classes:
      - HANCESTRO:0004
    label: Race
```

Will ask @dosumis to confirm.

Comment (ubyndr) [reply config/prod/neo4j2owl-config.yaml:214]: I guess you are right, that would also add the tag to the subclasses as well given the 'Race' example. I wasn't sure, thanks for the other example

Comment (dosumis) [reply config/prod/neo4j2owl-config.yaml:214]: A single class is a DL query - just with a named class.

**Review by dosumis — APPROVED** [review]

**Review by evanbiederstedt — APPROVED** [review]
Seems reasonable enough for me, based on what I can follow

**Pull request merged**